src/classes/repositories/class-tainacan-item-metadata.php

```diff
@@ -358,7 +358,7 @@ public function get_value( Entities\Item_Metadata_Entity $item_metadata ) {
 
 			global $wpdb;
 			$rows = $wpdb->get_results(
-				$wpdb->prepare( "SELECT * FROM $wpdb->postmeta WHERE post_id = %d AND meta_key = %s", $item_metadata->get_item()->get_id(), $item_metadata->get_metadatum()->get_id() ),
+				$wpdb->prepare( "SELECT * FROM $wpdb->postmeta WHERE post_id = %d AND meta_key = %s ORDER BY meta_id DESC", $item_metadata->get_item()->get_id(), $item_metadata->get_metadatum()->get_id() ),
```

Comment:
@rafaelchavesfreitas, to keep the consistent tests, I suggest changing the order to get always older metadata first:

`$wpdb->prepare( "SELECT * FROM $wpdb->postmeta WHERE post_id = %d AND meta_key = %s ORDER BY meta_id ASC", $item_metadata->get_item()->get_id(), $item_metadata->get_metadatum()->get_id() ),`